# Picked from: https://chromium.googlesource.com/chromium/src/+/29c1262dbd4f749dd514d05a571cdd809ba8cfe0
# Fix CUPS compilation error in OSX 10.12.
#
# BUG=622493
#
# Review-Url: https://codereview.chromium.org/2248343002
# Cr-Commit-Position: refs/heads/master@{#412670}

--- a/printing/backend/print_backend_cups.cc
+++ b/printing/backend/print_backend_cups.cc
@@ -4,6 +4,7 @@
 
 #include "printing/backend/print_backend_cups.h"
 
+#include <cups/ppd.h>
 #include <dlfcn.h>
 #include <errno.h>
 #include <pthread.h>
